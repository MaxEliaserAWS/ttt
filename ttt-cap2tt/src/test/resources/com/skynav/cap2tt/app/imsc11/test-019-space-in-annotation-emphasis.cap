Lambda字幕V4	DF0+1	SCENE"和文標準"

４	00005807/00005911	＠ルビ上［彼｜ ↓ ］＠と出会ったの
３８	00021220/00021416	いや＠ルビ上［任｜ ↓ ］＠＠ルビ上［務｜ ↓ ］＠＠ルビ上［で｜ ↓ ］＠ペルーだ
４４	00023101/00023212	ムダだ リ＠ルビ上［サ｜ ↓ ］＠
７７	00042909/00043120	オハイオの“州＠ルビ上［鳥｜ ↓ ］＠”だよ
７８	00043408/00043606	お前は “州＠ルビ上［ヤ｜ ↓ ］＠＠ルビ上［ロ｜ ↓ ］＠＠ルビ上［ウ｜ ↓ ］＠”だ
１６３	00110107/00110411	最近＠ルビ上［ご｜ ↓ ］＠＠ルビ上［無｜ ↓ ］＠＠ルビ上［沙｜ ↓ ］＠＠ルビ上［汰｜ ↓ ］＠なもんで
１８０	00122308/00122503	＠ルビ上［蝶｜ ↓ ］＠＠ルビ上［つ｜ ↓ ］＠＠ルビ上［が｜ ↓ ］＠＠ルビ上［い｜ ↓ ］＠になる
１２２９	01172618/01173022	“彼に抱かれた！
				＠ルビ下［早｜↑］＠＠ルビ下［か｜ ↑ ］＠＠ルビ下［っ｜ ↑ ］＠＠ルビ下［た｜ ↑ ］＠けどサイコー”
