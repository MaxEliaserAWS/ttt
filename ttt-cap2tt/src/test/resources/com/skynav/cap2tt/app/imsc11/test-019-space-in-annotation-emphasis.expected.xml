<?xml version="1.0" encoding="utf-8"?><tt xmlns="http://www.w3.org/ns/ttml" ttva:model="imsc11" xml:lang="ja" xmlns:tts="http://www.w3.org/ns/ttml#styling" xmlns:ttva="http://skynav.com/ns/ttv/annotations"><head><styling><initial tts:fontSize="4rh"/><initial tts:lineHeight="5rh"/><initial tts:showBackground="whenActive"/><style xml:id="s1" tts:textAlign="center"/><style xml:id="s2" tts:textAlign="start"/><style xml:id="s3" tts:textEmphasis="dot before"/><style xml:id="s4" tts:textEmphasis="dot after"/></styling><layout><region xml:id="横下" tts:displayAlign="after" tts:extent="80rw 15rh" tts:position="center bottom 10rh"/></layout></head><body region="横下" xml:space="preserve"><div><p begin="00:00:58:07" end="00:00:59:11" style="s1"><span style="s2"><span style="s3">彼</span>と出会ったの</span></p><p begin="00:02:12:20" end="00:02:14:16" style="s1"><span style="s2">いや<span style="s3">任</span><span style="s3">務</span><span style="s3">で</span>ペルーだ</span></p><p begin="00:02:31:01" end="00:02:32:12" style="s1"><span style="s2">ムダだ リ<span style="s3">サ</span></span></p><p begin="00:04:29:09" end="00:04:31:20" style="s1"><span style="s2">オハイオの“州<span style="s3">鳥</span>”だよ</span></p><p begin="00:04:34:08" end="00:04:36:06" style="s1"><span style="s2">お前は “州<span style="s3">ヤ</span><span style="s3">ロ</span><span style="s3">ウ</span>”だ</span></p><p begin="00:11:01:07" end="00:11:04:11" style="s1"><span style="s2">最近<span style="s3">ご</span><span style="s3">無</span><span style="s3">沙</span><span style="s3">汰</span>なもんで</span></p><p begin="00:12:23:08" end="00:12:25:03" style="s1"><span style="s2"><span style="s3">蝶</span><span style="s3">つ</span><span style="s3">が</span><span style="s3">い</span>になる</span></p><p begin="01:17:26:18" end="01:17:30:22" style="s1"><span style="s2">“彼に抱かれた！<br/><span style="s4">早</span><span style="s4">か</span><span style="s4">っ</span><span style="s4">た</span>けどサイコー”</span></p></div></body></tt>